index.html

```diff
@@ -49,5 +54,204 @@ <h4>I’am a Software Developer</h4>
+          See Projects         
+        </button>
+      </div>
+   </section>
+
+   <section class="Projects"> 
+    <div class="first-project"> 
+        <h1>Profesional Art  <br> Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#">
+          See Project
+        </a>
+      </div>
+
+      <div class="first-project"> 
+        <h1>Profesional Art  <br> Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#">
+          See Project
+        </a>
+      </div>
+
+      <div class="first-project"> 
+        <h1>Profesional Art  <br> Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#">
+          See Project
+        </a>
+      </div>
+
+      <div class="first-project"> 
+        <h1>Profesional Art  <br> Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#">
+          See Project
+        </a>
+      </div>
+
+      <div class="first-project"> 
+        <h1>Profesional Art 
+           <br> Printing Data
+        </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#">
+          See Project
+        </a>
+      </div>
+
+      <div class="first-project"> 
+        <h1>Profesional Art  
+          <br> Printing Data
+        </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#">
+          See Project
+        </a>
+      </div>
+   </section>
+
+  <section class="about-me">
+   <div class="about">
+    <h1>About me</h1>
+    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <button type="button">
+      Get My Resume
+    </button>
```

Comment:
- [ ] Your button should follow the button styling: Active, Hover and Pressed as provided from Figma design. This will make your webpage more interactive and accessible.